src/features/offer/helpers/useHasEnoughCredit/useHasEnoughCredit.test.ts

```diff
@@ -15,12 +17,11 @@ jest.mock('features/offer/api/useOffer', () => ({
   }),
 }))
 
-let mockedUser: Partial<UserProfileResponse> | undefined = undefined
-jest.mock('features/profile/api', () => ({
-  useUserProfileInfo: () => ({
-    data: mockedUser,
-  }),
-}))
+const mockUseAuthContext = jest.spyOn(Auth, 'useAuthContext').mockReturnValue({
+  user: undefined,
+  isLoggedIn: false,
+  setIsLoggedIn: jest.fn(),
+})
```

Comment:
C'est pas le même mock qu'avant, c'est normal ?

Surtout qu'il y a un mockReturnValueOnce avec la même valeur dans le test `useHasEnoughCredit > should return false if no offer nor user found`